pages/mesosphere/dcos/version-policy/index.md

```diff
@@ -83,98 +81,97 @@ Customers running DC/OS on non-supported platform components should upgrade to a
 <table class="table">
     <tr>
     <th><strong>Platform Component</strong></th>
+    <th><strong>DC/OS 2.1 Latest Stable</strong></th>
     <th><strong>DC/OS 2.0 Latest Stable</strong></th>
     <th><strong>DC/OS 1.13 Latest Stable</strong></th>
-    <th><strong>DC/OS 1.12 Latest Stable</strong></th>
     </tr>
     <tr>
-        <td>RHEL 8.0</td>
+        <td>RHEL 8.*</td>
         <td><p style="text-align: center;">Docker CE 18.09.1</p></td>
         <td><p style="text-align: center;">Docker CE 18.09.1</p></td>
         <td><p style="text-align: center;">Docker CE 18.09.1</p></td>
     </tr>
     <tr>
-        <td>RHEL 7.7</td>
-        <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
-        <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
-        <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
-    </tr>
-    <tr>
-        <td>RHEL 7.6</td>
+        <td>RHEL 7.*</td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
         <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
     </tr>
 </table>
 
 ### <a name="Ubuntu-support-matrix"></a>Ubuntu support matrix
+*NOTE: Starting with DC/OS 2.0, we have discontinued support for Ubuntu Linux and have ceased qualifying it as an Operating System*.
```

Comment:
That phrase sounds a bit strange to me, because that it may sound as we qualify what is an operating system and what is not. I would simply say:
> *NOTE: Starting with DC/OS 2.0, we have discontinued support for Ubuntu Linux.*

Reply:
thanks!